packages/app/src/config.ts

```diff
@@ -11,10 +9,12 @@ export const CLICKHOUSE_HOST =
 
 // ONLY USED IN LOCAL MODE
 // ex: HDX_LOCAL_DEFAULT_CONNECTIONS='[{"id":"local","name":"Demo","host":"https://demo-ch.hyperdx.io","username":"demo","password":"demo"}]' HDX_LOCAL_DEFAULT_SOURCES='[{"id":"l701179602","kind":"trace","name":"Demo Traces","connection":"local","from":{"databaseName":"default","tableName":"otel_traces"},"timestampValueExpression":"Timestamp","defaultTableSelectExpression":"Timestamp, ServiceName, StatusCode, round(Duration / 1e6), SpanName","serviceNameExpression":"ServiceName","eventAttributesExpression":"SpanAttributes","resourceAttributesExpression":"ResourceAttributes","traceIdExpression":"TraceId","spanIdExpression":"SpanId","implicitColumnExpression":"SpanName","durationExpression":"Duration","durationPrecision":9,"parentSpanIdExpression":"ParentSpanId","spanKindExpression":"SpanKind","spanNameExpression":"SpanName","logSourceId":"l-758211293","statusCodeExpression":"StatusCode","statusMessageExpression":"StatusMessage"},{"id":"l-758211293","kind":"log","name":"Demo Logs","connection":"local","from":{"databaseName":"default","tableName":"otel_logs"},"timestampValueExpression":"TimestampTime","defaultTableSelectExpression":"Timestamp, ServiceName, SeverityText, Body","serviceNameExpression":"ServiceName","severityTextExpression":"SeverityText","eventAttributesExpression":"LogAttributes","resourceAttributesExpression":"ResourceAttributes","traceIdExpression":"TraceId","spanIdExpression":"SpanId","implicitColumnExpression":"Body","traceSourceId":"l701179602"}]' yarn dev:local
-export const HDX_LOCAL_DEFAULT_CONNECTIONS =
-  publicRuntimeConfig.hdxLocalDefaultConnections;
-export const HDX_LOCAL_DEFAULT_SOURCES =
-  publicRuntimeConfig.hdxLocalDefaultSources;
+export const HDX_LOCAL_DEFAULT_CONNECTIONS = env(
+  'NEXT_PUBLIC_HDX_LOCAL_DEFAULT_CONNECTIONS',
+);
+export const HDX_LOCAL_DEFAULT_SOURCES = env(
```

Comment:
I feel it would be handy if the `NEXT_PUBLIC_IS_LOCAL_MODE` is also runtime

Reply:
I'll leave that to a separate PR, since I think we do different images/serving modes for that anyways.